docker-compose.yml

```diff
@@ -33,3 +33,15 @@ services:
       - DATABASE_URL=postgres://dev-node:insecure-change-me-in-prod@postgres:5432/dev-node?sslmode=disable
     depends_on:
       - postgres
+  database:
```

Comment:
We might want to start having multiple different files, and `up.sh` would take the correct identifier. We can do it later also.